jac-uk/admin
# #2165 Bug: Exercise. Eligibility Information. ASC word limit messaging is illogical

## What's included?
Ensure the field specifying the word limit is not a required field and the default word limit applies when not filled in.

Closes #2165 

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?

**There is a video attached below which shows the whole testing process!**

Test 1:
- Go to the following page (you may need to login first): https://jac-admin-develop--pr2218-feature-2165-asc-wor-srzka530.web.app/exercise/Vulmahzn4VtelQhVlJy6/details/overview
- You may need to unlock the exercise for editing
- Go to the Eligibility Information section and press 'Update eligibility information'
- Scroll down to the section which says: Does additional selection criteria (ASC) apply?
- Select 'No' and then 'Save And Continue'. If there are any errors just click the 'Save And Continue Anyway' button

Test 2
- At the top of the page click 'Update eligibility information'
- Scroll down to the section which says: Does additional selection criteria (ASC) apply?
- Select 'Yes'
- Add a title and text **but no word limit**
- Click the button to 'add another'
- Add a title and text **and specify a word limit of 10**
- Click 'Save And Continue'. If there are any errors just click the 'Save And Continue Anyway' button

Test 3
- At the top of the page click 'Update eligibility information'
- Scroll down to the section which says: Does additional selection criteria (ASC) apply?
- Ensure the titles/text/word counts are as entered in the previous test

**Now you can approve and publish the exercise then go to the Apply site and follow the instructions in this pull request:** 

## Risk - how likely is this to impact other areas?
🟢 No risk - this is a self-contained piece of work

## Additional context

https://github.com/jac-uk/admin/assets/115651787/11be1b8c-bb71-4f52-b920-987cc618b306

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
